lib/elixir/lib/code.ex

```diff
@@ -1249,11 +1248,22 @@ defmodule Code do
     file = Keyword.get(opts, :file, "nofile")
     line = Keyword.get(opts, :line, 1)
     column = Keyword.get(opts, :column, 1)
+    include_comments = Keyword.get(opts, :include_comments, false)
+
+    Process.put(:code_formatter_comments, [])
+    opts = [preserve_comments: &preserve_comments/5] ++ opts
```

Comment:
I think I told you privately that we could make `preserve_comments: :ast` an option but I now realized that `preserve_comments` is actually a private option, so the `:include_comments` option is better.